src/client/protocolgame.cpp

```diff
@@ -33,7 +33,13 @@ void ProtocolGame::login(const std::string_view accountName, const std::string_v
     m_sessionKey = sessionKey;
     m_characterName = characterName;
 
+#ifndef __EMSCRIPTEN__
     connect(host, port);
+#else
+    if (port == 7172)
```

Comment:
what's magic about 7172?

Reply:
Being the default game world port, it means that the user trying to use the web client didn't properly enforce the correct port to be used with websockets, so it's redirect to 443 for ease of use.